### ISSUE TITLE
Error when opening `pubspec.yaml`

### ISSUE THREAD

**Issue by sidlatau** [akinsho/pubspec-assist.nvim#3]
I am getting an error when opening pubspec.yaml:
```
Error executing vim.schedule lua callback: .../packer/start/pubspec-assist.nvim/lua/pubspec-assist.lua:80: attempt to compare nil with number                            
stack traceback:                                                                                                                                                         
        .../packer/start/pubspec-assist.nvim/lua/pubspec-assist.lua:80: in function 'is_greater'                                                                         
        .../packer/start/pubspec-assist.nvim/lua/pubspec-assist.lua:107: in function 'get_package_state'                                                                 
        .../packer/start/pubspec-assist.nvim/lua/pubspec-assist.lua:115: in function 'add_version'                                                                       
        .../packer/start/pubspec-assist.nvim/lua/pubspec-assist.lua:214: in function 'on_success'                                                                        
        .../packer/start/pubspec-assist.nvim/lua/pubspec-assist.lua:152: in function <.../packer/start/pubspec-assist.nvim/lua/pubspec-assist.lua:147>                   
                
```

**Comment by akinsho** [comment]
Hi @sidlatau,

Thanks for reporting this. I recently refactored this plugin to use a different yaml parser because installing the `lyaml` luarock is a pain if not using packer. I thought this would be easier to use/maintain since it's just a one file I copied into the repo.

It must have a bug though, since it's not a huge project, I just tried it on two different pubspecs I had available. Can you link me the pubpec you are using, so I can try it locally? 
If it's private, could you maybe paste a copy of the values at line 80 in the file in the stack trace?

**Comment by sidlatau** [comment]
I investigated a bit more, looks like the problem appears when parsing line: `  intl: any`

**Comment by akinsho** [comment]
Hmm I just tried adding `intl: any` to a pubspec and nothing happened. Where exactly are you putting it inside your pubspec yaml.

**Comment by sidlatau** [comment]
It does not matter for me where it is in the file. Here is not working example:
```yml
dependencies:
  flutter:
    sdk: flutter
  flutter_localizations:
    sdk: flutter
  intl: any # Use the pinned version from flutter_localizations
  url_launcher: ^6.1.8

```
I created a PR that fixes this problem for me #4
